tokio-trace/tokio-trace-core/src/callsite.rs

```diff
@@ -67,24 +88,21 @@ pub struct Identifier(
     pub &'static Callsite,
 );
 
+/// Clear and reregister interest on every [`Callsite`]
+///
+/// [`Callsite`]: ../callsite/trait.Callsite.html
+pub fn invalidate_cache() {
```

Comment:
Super nitpicky, but unpon thinking about this a bit longer, why don't we call this 
```suggestion
pub fn invalidate_interest() {
```
rather than `invalidate_cache`? That way, the name makes it clearer what's being invalidated.